docs/identities/sign-in/check-session.mdx

```diff
@@ -0,0 +1,150 @@
+title: Check for user session status in browser cookies and session tokens
+sidebar_label: Check login session
+slug: check-session-token-cookie-api
+---
+
+# Check session status
+
+import Tabs from "@theme/Tabs"
+import TabItem from "@theme/TabItem"
+import { getSdkUrl } from "@site/src/hooks"
+import CodeBlock from "@theme/CodeBlock"
+import CodeFromRemote from "@theme/CodeFromRemote"
+import expressjs from "!!raw-loader!../../../code-examples/identity/sign-in/check-session/js/server.js"
+import ajax from "!!raw-loader!../../../code-examples/identity/sign-in/check-session/js/client.js"
+import goExample from "!!raw-loader!../../../code-examples/identity/sign-in/check-session/go/server.go"
+
+Ory stores session data in different ways for browser apps and native apps:
+
+- For browser apps, Ory stores session data in Ory Session Cookies
+- For native apps, Ory stores session data in Ory Session Tokens
+
+:::tip
+
+Refresh your knowledge on the [cookie-based security](../../security-model.mdx) used by Ory and
+[the difference between browser and native APIs](../native-browser.mdx).
+
+:::
+
+To check if the user is signed in and has an active session, use the `/sessions/whoami` endpoint. If the user doesn't have an
+active session, the system returns a `401 Unauthorized` response. If the user has an active session, you get a `200 OK` response
+and the session payload.
+
+When using the SDK, use the `frontend.toSession()` method.
+
+````mdx-code-block
+
+<Tabs>
+<TabItem value="curl" label="cURL" default>
+
+To check for an active session with an Ory Session Token, run:
+
+```shell
+curl -H "Authorization: Bearer {your-session-token}" \
+  "https://{project.slug}.projects.oryapis.com/sessions/whoami"
+```
+
+To check for an active session with an Ory Session Cookie, run:
+
+```shell
+curl -H "Cookie: ory_session_...=..." \
+  "https://{project.slug}.projects.oryapis.com/sessions/whoami"
+```
+
+</TabItem>
+<TabItem value="ajax" label="AJAX / SPA">
+
+When calling Ory or your API from a Single Page Application (SPA) using AJAX:
+
+- The SPA (`www.yourdomain.com`, `localhost:3000`), Ory (`ory.yourdomain.com`, `localhost:4000`), and your API must be on the same domain.
+- You configured CORS in the Ory Console and on your API server.
+- The cookie is configured to be available on the domains you need, for example, the full `yourdomain.com`.
+
+<CodeBlock className="language-js">{ajax}</CodeBlock>
+
+</TabItem>
+<TabItem value="expressjs" label="ExpressJS">
+
+Ory checks for both the cookie and the token, which means that your API must handle both Ory Session Cookies and Ory Session Tokens:
+
+<CodeBlock className="language-js">{expressjs}</CodeBlock>
+
+</TabItem>
+<TabItem value="go" label="Go">
+
+Ory checks for both the cookie and the token, which means that your API must handle both Ory Session Cookies and Ory Session Tokens:
+
+<CodeBlock className="language-go">{goExample}</CodeBlock>
+
+</TabItem>
+</Tabs>
+````
+
+## Session response
```

Comment:
There's more about sessions (including a full session example with some explainers) here: 
https://www.ory.sh/docs/kratos/session-management/overview#ory-session

Maybe instead of duplicating this content, it'd be better to point the user toward this set of docs?

Reply:
Hm we could probably create a file and import it then across the different pages. Since the sessions section has not been touched yet, i would probably keep it here for now, but we can have a follow up issue to clean it up!